test/extended/testdata/service_catalog/ups-deployment.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ups-deployment-template
+objects:
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    labels:
+      app: ups-broker
+      chart: ups-broker-0.0.1
+      heritage: Tiller
+      release: ups-broker
+    name: ups-broker
+    namespace: "${NAMESPACE}"
```

Comment:
just in case, can you use a different name for this parameter in case somehow things are being tripped by NAMESPACE being a special value?

i don't think it's the issue but it's an easy thing to try.